crates/holochain_types/src/zome_types.rs

```diff
@@ -14,9 +14,14 @@ mod error;
 #[cfg(test)]
 mod test;
 
+/// TODO
+pub type NumZomeTypes = NonZeroU8;
```

Comment:
Why should this be nonzero? A zome may have no entry or link types.

Reply:
We don't store dependencies that don't have zero types. This is important for the scoped zome types to work.